instr.ml

```diff
@@ -194,43 +153,36 @@ let expr_vars = function
     List.map simple_expr_vars xs
     |> List.fold_left VarSet.union VarSet.empty
 
-let arg_vars = function
-  | Arg_by_val e -> expr_vars e
-  | Arg_by_ref x -> VarSet.singleton x
+let arg_vars = expr_vars
```

Comment:
Again, I feel like `arg_vars` is used in so few places that we might as well change it to `exp_vars`.